java/code/src/com/suse/manager/webui/controllers/admin/AdminViewsController.java

```diff
@@ -136,4 +140,30 @@ public static ModelAndView showPayg(Request request, Response response, User use
         }
         return new ModelAndView(data, "controllers/admin/templates/payg.jade");
     }
+
+    /**
+     * Show proxy tab.
+     * @param request http request
+     * @param response http response
+     * @param user current user
+     * @return the view to show
+     */
+    public static ModelAndView showProxy(Request request, Response response, User user) {
+        Map<String, Object> data = new HashMap<>();
+        ProxySettingsDto proxySettings = ProxySettingsManager.getProxySettings();
+        data.put("proxySettings", GSON.toJson(proxySettings));
+
+        /**
+         * TODO: Would also love to data.put() whether the proxy is verified similar to
```

Comment:
@parlt91 If you have some spare cycles at some point, I have two todos here that I could use some help with please. Hopefully they're both pretty self explanatory, I want to pass some additional data to the frontend, but I couldn't figure out what the right way to extract that data is with the current logic that's already there.

Reply:
Fyi this is not urgent in any way, so feel free to let me know at your convenience.

Reply:
The password bit is not a hard requirement, would be nice from a usability point of view, but of course I can see your side of it as well. It's not a must have.

Reply:
As we discussed on the call, the async behavior works better here, so I'll be reverting this change.